src/main.js

```diff
@@ -17,6 +17,7 @@ async function run() {
     const coverageFilesPattern = core.getInput('coverage-files');
     const globber = await glob.create(coverageFilesPattern);
     const coverageFiles = await globber.glob();
+    const title = core.getInput('title');
```

Comment:
@zgosalvez While testing my PR, I've discovered that `title` is used here, but the parameter is called `title-prefix` in https://github.com/jhen0409/github-actions-report-lcov/blob/c949a663a9050ce84129e34888d7f40116ff2f56/action.yml#L19

Reply:
Thank you! I've released hotfixes for v1 and v2.